app/config/config.yml

```diff
@@ -43,7 +45,7 @@ framework:
     session:
         # handler_id set to null will use default session handler from php.ini
         handler_id:  ~
-    fragments:       ~
+    fragments:       { path: /_fragment }
     http_method_override: true
```

Comment:
@sergiu-popa, this change isn't needed. See https://github.com/symfony/symfony/blob/master/src/Symfony/Bundle/FrameworkBundle/DependencyInjection/Configuration.php#L175